Jenkinsfile

```diff
@@ -226,13 +216,23 @@ def job = {
                                 sh cmd
                             }
                             step([$class: 'hudson.plugins.findbugs.FindBugsPublisher', pattern: '**/*bugsXml.xml'])
+
+                            if (!config.isPrJob) {
+                                def git_tag = "v${config.docker_tag}-ksqldb"
+                                sh "git add ."
+                                sh "git commit -m \"build: Setting project version ${config.ksql_db_artifact_version} and parent version ${config. cp_version}.\""
```

Comment:
i don't know if this is just a display problem with the diff, but it seems like you have a space between "${config." and "cp_version}".

Reply:
Good catch -- fixed.